src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Hooks.java

```diff
@@ -1,147 +1,166 @@
 package gov.healthit.chpl.aqa.stepDefinitions;
 
-import static io.restassured.RestAssured.given;
-
 import java.io.File;
+import java.io.FileInputStream;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.util.Date;
 import java.util.HashMap;
+import java.util.Properties;
 import java.util.concurrent.TimeUnit;
-
-import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang3.StringUtils;
-import org.junit.Test;
 import org.openqa.selenium.OutputType;
 import org.openqa.selenium.TakesScreenshot;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.io.FileHandler;
 import org.openqa.selenium.remote.CapabilityType;
-import org.openqa.selenium.remote.DesiredCapabilities;
 import org.openqa.selenium.support.events.AbstractWebDriverEventListener;
-//import org.openqa.selenium.chrome.ChromeOptions;
 import org.openqa.selenium.support.events.EventFiringWebDriver;
 import org.openqa.selenium.support.events.WebDriverEventListener;
-
 import cucumber.api.java.After;
 import cucumber.api.java.Before;
-import io.restassured.RestAssured;
-import io.restassured.http.ContentType;
-import io.restassured.path.json.JsonPath;
-import io.restassured.response.Response;
 
 /**
  * Class Hooks definition.
  */
 public class Hooks {
 
-    private static File dir;
-    private static EventFiringWebDriver driver;
-    private static final int DELAY = 30;
-    private static String screenshotPath;
-    private static String downloadPath = System.getProperty("downloadPath");
-
-    /**
-     * Launch ChromeDriver.
-     */
-    @Before("~@RegressionAPI")
-    public void openBrowser() {
-        /* To run chrome with the developer tools window automatically opened re-able these lines.
-         *
+	private static File dir;
+	private static EventFiringWebDriver driver;
+	public static Properties config = new Properties();
+	private static final int DELAY = 30;
+	private static String screenshotPath;
+	public static FileInputStream fis;
+	private static String downloadPath = System.getProperty("downloadPath");
+	public static String browser;
+	/**
+	 * Launch ChromeDriver.
+	 * @throws IOException 
+	 */
+	@Before("~@RegressionAPI")
+	public void openBrowser() throws IOException {
+		/* To run chrome with the developer tools window automatically opened re-able these lines.
```

Comment:
These lines probably need to be moved to the chrome section of the method; they won't work for firefox, right?